specificatie/verblijfplaats.yaml

```diff
@@ -0,0 +1,435 @@
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            datumVestigingInNederland:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            gemeenteVanInschrijving:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+            landVanwaarIngeschreven:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            inOnderzoek:
+              $ref: "#/components/schemas/VerblijfplaatsInOnderzoek"
+    Locatie:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaats'
+        - type: object
+          properties:
+            adresregel1:
+              $ref: '#/components/schemas/Adresregel1'
+            adresregel2:
+              $ref: '#/components/schemas/Adresregel2'
+            datumInschrijvingInGemeente:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            datumVestigingInNederland:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            functieAdres:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            gemeenteVanInschrijving:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+            indicatieVestigingVanuitBuitenland:
+              $ref: '#/components/schemas/IndicatieVestigingVanuitBuitenland'
+            landVanwaarIngeschreven:
+              $ref: 'https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel'
+            locatiebeschrijving:
+              $ref: '#/components/schemas/Locatiebeschrijving'
+            vanuitVertrokkenOnbekendWaarheen:
+              $ref: '#/components/schemas/VanuitVertrokkenOnbekendWaarheen'
+            woonplaats:
+              $ref: '#/components/schemas/Woonplaats'
+            inOnderzoek:
+              $ref: '#/components/schemas/LocatieInOnderzoek'
+    VerblijfplaatsOnbekend:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaats'
+        - type: object
+          properties:
+            datumInschrijvingInGemeente:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            gemeenteVanInschrijving:
+              $ref: '#/components/schemas/GemeenteVanInschrijving'
+            vertrokkenOnbekendWaarheen:
+              $ref: '#/components/schemas/VertrokkenOnbekendWaarheen'
+            inOnderzoek:
+              $ref: '#/components/schemas/VerblijfplaatsOnbekendInOnderzoek'
+    GbaVerblijfplaats:
+      allOf:
+        - $ref: '#/components/schemas/AbstractVerblijfplaats'
```

Comment:
doordat ook GbaVerblijfplaats overerft uit AbstractVerblijfplaats, wordt bij Persoon (response bij raadplegen op bsn) ook GbaVerblijfplaats als optie getoond in Redoc. Bij AbstractVerblijfplaatsBeperkt gebeurt dat niet (wordt GbaVerblijfplaatsBeperkt niet getoond).
Andersom wordt ook bij GbaPersoon de optie voor alle verblijfplaatssoorten getoond, niet alleen GbaVerblijfplaats.
Kun je dit zo maken dat bij GbaPersoon alleen GbaVerblijfplaats mogelijk is en bij Persoon GbaVerblijfplaats niet mogelijk is?

Reply:
done